Add Vampire Diaries API

afaf98 opened the pull request:
<!-- Thank you for taking the time to work on a Pull Request for this project! -->
<!-- To ensure your PR is dealt with swiftly please check the following: -->
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

P.S. I Read the contributing guidelines as carefully as I could, this my first open source PR, if something needs to change I would love feedback. Thank you for this awesome project.

PawelBorkar commented:
Hey @afaf98 welcome to the open source community.
It seems that the link is broken after pasting the API key and query it's not returning the data. Check your herokuapp hosting.

afaf98 commented:
Hey @PawelBorkar,

Thank you for your feedback. I improved my Api explorer in the following way : 

- I disabled making a request, until you've pasted your apiKey
- I added some default parameters for route and query
- When you paste the key you should get a success response.

You can try it out [here](https://vampire-diaries-api.netlify.app/)

I've added some screenshots below :

#### No apiKey
![Screenshot from 2021-03-30 11-31-24](https://user-images.githubusercontent.com/44358838/112969214-546ef800-914d-11eb-8257-bda9a09eb73b.png)

#### With apiKey

![Screenshot from 2021-03-30 11-31-39](https://user-images.githubusercontent.com/44358838/112969309-69e42200-914d-11eb-9d93-a5396db2f4fd.png)

(The apiKey on the screenshot is in the development database :) )

PawelBorkar commented:
Thanks for your contribution @afaf98 :)
and a quick suggestion for the website the Route and Query fields are prefilled the placeholders looks good just remove the prefilled data from Route and Query fields.
